docs/contributor/adr/0001-istio-jwt-proposal.md

```diff
@@ -0,0 +1,79 @@
+# Istio JWT Proposal
+
+Date: 2022-02-03
+
+## Status
+
+- Accepted (2022-12-29)
```

Comment:
I would also only list the most recent status without a date. We will have the other info as part of the version control.